modules/globebrowsing/globebrowsingmodule_lua.inl

```diff
@@ -28,27 +28,32 @@
 namespace {
 
 /**
- * Adds a layer to the specified globe. The first argument specifies the name of the scene
- * graph node of which to add the layer. The renderable of the specified scene graph node
- * needs to be a renderable globe. The second argument is the layer group which can be any
+ * Adds a layer to the specified globe. The second argument is the layer group which can be any
  * of the supported layer groups. The third argument is the dictionary defining the layer.
+ *
+ * \param globeIdentifier The identifier of the scene graph node of which to add the
+ *                        layer. The renderable of the scene graph node must be a
+ *                        [RenderableGlobe](#globebrowsing_renderableglobe)
+ * \param layerGroupName The name of the layer group in which to add the layer
```

Comment:
I guess this would be an identifier as well then?  We don't really distinguish them, but since the layer groups are a properyowner